backend/cognito/jest.config.ts

```diff
@@ -0,0 +1,7 @@
+import {Config} from "jest";
+import baseConfig from "../../jest.config.base";
+
+export default {
+    ...baseConfig,
+    displayName: "api"
```

Comment:
```suggestion
    displayName: "cognito"
```

Reply:
You also need to add a `tsconfig.json` to the `tests` directory (can be copied from `api/tests`)

Reply:
addressed